src/client/components/componentWithSliders/allSliders/blockOfSliders/slider/slider.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Slider from 'react-rangeslider';
+import 'react-rangeslider/lib/index.css';
+
+import './slider.css';
+
+class OneSlider extends Component {
+  constructor (props, context) {
+    super(props, context);
+    this.state = {
+      value: props.value,
```

Comment:
its better to call your variables more informative, to know what is it when you read code, for eg you have `value` here, but what value?